lib/resources/crontab.rb

```diff
@@ -25,51 +25,35 @@ class Crontab < Inspec.resource(1)
       describe crontab.where { command =~ /a partial command string/ } do
         its('entries.length') { should cmp 1 }
       end
+
+      describe crontab({path: '/etc/cron.d/some_crontab'}) do
+        its('commands') { should include '/path/to/some/script' }
+      end
     "
 
     attr_reader :params
 
     include CommentParser
 
-    def initialize(user = nil)
-      @user   = user
+    def initialize(opts = {})
+      Hash[opts.map { |k, v| [k.to_sym, v] }] if opts.respond_to?(:fetch)
+      @user = opts.respond_to?(:fetch) ? opts.fetch(:user, nil) : opts
+      @path = opts.fetch(:path, nil) if opts.respond_to?(:fetch)
       @params = read_crontab
 
       return skip_resource 'The `crontab` resource is not supported on your OS.' unless inspec.os.unix?
     end
 
     def read_crontab
-      inspec.command(crontab_cmd).stdout.lines.map { |l| parse_crontab_line(l) }.compact
+      ct = @path.nil? ? inspec.command(crontab_cmd).stdout : inspec.file(@path).content
```

Comment:
We do the `@path.nil?` check a few times. What do you think about adding a helper method or to?

```ruby
def is_system_crontab?
  !@path.nil?
end

def is_user_crontab?
  !@user.nil?
end
```

We could use that here, in the `to_s` method, and in `parse_crontab_line`.